conda-recipes/cyclus-deps/post-link.sh

```diff
@@ -1,30 +1,3 @@
-#/bin/bash
```

Comment:
Ah (it is mostly being delted, not wholy, FWIW). This looks to have been
directly copied from the cyclus conda recipe. It assumes that cyclus was
actually installed (hence playing with the executable name). Cyclus-deps
does *not* install cyclus.

On Wed, Jun 3, 2015 at 10:40 AM, Paul Wilson <notifications@github.com>
wrote:

> In conda-recipes/cyclus-deps/post-link.sh
> <https://github.com/cyclus/release/pull/72#discussion_r31636783>:
>
> > @@ -1,30 +1,3 @@
> > -#/bin/bash
>
> I was wondering about the whole file being deleted but just tagged it on
> this line.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/release/pull/72/files#r31636783>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192